### DISCUSSION TITLE
请问一下 我用github建设的项目运行完毕以后  访问网址 提示这个

### DISCUSSION THREAD

**Discussion by bloy123** [Guovin/iptv-api#774] [Q&A] [answered]
按照您的步骤  我只有自定义频道那里没有改 其他都是按照您的流程一步一步走的

这个结果    更新时间,#genre#
2024-12-31 15:52:23,url           是更新成功了吗？ 我放到TVBOX里面 解析不出来。

**Comment by Guovin** [comment, marked as answer]
#699 可以看下这个，tvbox部分版本不支持最新接口，建议使用其它播放器
